src/foam/nanos/notification/NotificationSetting.js

```diff
@@ -67,21 +67,8 @@ foam.CLASS({
         { name: 'notification', type: 'foam.nanos.notification.Notification' }
       ],
       javaCode: `
-        X userX = x;
```

Comment:
@lchanmann why are you removing this? 
I intentionally added it recently, so all downstream - from notificationSettings - logic didn't have to worry about this.

Reply:
Reverted in https://github.com/foam-framework/foam2/pull/4617/commits/816722dda2f442fd04c5ed9425fe5917936c42b5.